File: INICIANTE/1012.js
let A = Number(lines.shift());
let B = Number(lines.shift());
let C = Number(lines.shift());
let areaTrianguloR = ((A*C)/2).toFixed(3);
let areaCirculo = (3.14159 * (C**2)).toFixed(3);
let areaTrapezio = (((A + B)*C)/2).toFixed(3);
let areaQuadrado = (B**2).toFixed(3);
let areaRetangulo = (A*B).toFixed(3);

console.log("TRIANGULO: "+areaTrianguloR+"\nCIRCULO: "+areaCirculo+"\nTRAPEZIO: "+ areaTrapezio+"\nQUADRADO: "+areaQuadrado+"\nRETANGULO: "+areaRetangulo);

File: INICIANTE/py/1012.py
A, B, C = map(float, input().split())
A_triangulo = A*C/2
A_circulo = 3.14159*C**2
A_trapezio = (A+B)*C/2
A_quadrado = B**2
A_retangulo = A*B
print(f"TRIANGULO: {A_triangulo:.3f}\nCIRCULO: {A_circulo:.3f}\nTRAPEZIO: {A_trapezio:.3f}\nQUADRADO: {A_quadrado:.3f}\nRETANGULO: {A_retangulo:.3f}")